apps/sms/js/thread_ui.js

```diff
@@ -158,7 +158,9 @@ var ThreadUI = {
     );
```

Comment:
Agree, in conversation view  we should reduce the access to Threads service(we still need to fetch thread data while panel entering, but after that I think we can cache it in local).